Mathlib/FieldTheory/SplittingField/IsSplittingField.lean

```diff
@@ -46,18 +46,27 @@ namespace Polynomial
 
 variable [Field K] [Field L] [Field F] [Algebra K L]
 
-/- ./././Mathport/Syntax/Translate/Command.lean:394:30: infer kinds are unsupported in Lean 4: #[`Splits] [] -/
-/- ./././Mathport/Syntax/Translate/Command.lean:394:30: infer kinds are unsupported in Lean 4: #[`adjoin_roots] [] -/
 /-- Typeclass characterising splitting fields. -/
 class IsSplittingField (f : K[X]) : Prop where
-  Splits : Splits (algebraMap K L) f
-  adjoin_roots : Algebra.adjoin K (↑(f.map (algebraMap K L)).roots.toFinset : Set L) = ⊤
+  splits' : Splits (algebraMap K L) f
+  adjoin_roots' : Algebra.adjoin K (↑(f.map (algebraMap K L)).roots.toFinset : Set L) = ⊤
 #align polynomial.is_splitting_field Polynomial.IsSplittingField
 
-variable {K L F}
-
 namespace IsSplittingField
 
+variable {K}
+
+-- Porting note: infer kinds are unsupported
```

Comment:
What do you mean by these porting notes? Could you please expand them a bit?

Reply:
I just lifted them from #4339. Do I need to add "in Lean 4"?

Reply:
I just don't know what these lines mean. What breaks if you remove them? Maybe that helps with writing a more useful porting note?

Reply:
The issue is that the f argument becomes implicit

Reply:
Aha, got it.

Reply:
```suggestion
-- Porting note: infer kinds are unsupported
-- so we provide a version of `splits'` with `f` explicit.
```